### PR TITLE
Ws week picker preact

### PR DESCRIPTION
**Pull request by fragsalat**
@schluchter  I re-created  #42  for you with the correct target branch :)

### CODE REVIEW

File: webcomponents/ws-date-picker/ws-date-picker.scss [hunk 1]
```diff
@@ -0,0 +1,211 @@
+$export-form-css: false;
```
Comment (fragsalat) [inline webcomponents/ws-date-picker/ws-date-picker.scss]: This file is definitely in the wrong folder. The folder webcomponents doesn't exists any more

**Comment by schluchter** [comment]
@fragsalat Removed webcomponents folder as requested.

**Comment by schluchter** [comment]
Build fails because of #44. Going to look at it now.

**Comment by schluchter** [comment]
@faboweb I made all necessary changes from my point of view. Can you approve the merge?

**Comment by fragsalat** [comment]
:+1: approved

**Comment by schluchter** [comment]
👍  super-approved

**Comment by faboweb** [comment]
There was no image of the changes like agreed on :(

**Pull request merged**